build-custom-image/README.md

```diff
@@ -0,0 +1,22 @@
+## Overview
+[Batch](https://cloud.google.com/batch/docs/get-started) has provided Batch images under project `batch-custom-image` for different OS types.
+The Batch images pre-installs the required packages for running Batch jobs, which can help you save Batch job running time, as well as overcoming network limitations.
+
+This folder provides strategy on how to build a Batch required (preferred) image with different OS types. Due to the latency on GitHub update, if you meet with any issues when running the scripts in the folder, feel free to get help from Batch team.
```

Comment:
"This folder contains instructions on how to create a Batch compatible image with various operating system types."

Remove "Due to ... get help from Batch team". We don't have an official channel for that.

Reply:
Modified, thanks!